internal/awss3/storage.go

```diff
@@ -100,8 +100,11 @@ func (s *Storage) traverse(ctx context.Context, repoURI string, items chan<- Cha
 				// is flat and cannot contain nested directories.
 				continue
 			}
-			if key == "index.yaml" {
-				// Ignore the index itself.
+
+			if !strings.Contains(key, ".tgz") {
```

Comment:
Wouldn't it be more correct?
```suggestion
			if !strings.HasSuffix(key, ".tgz") {
```

Reply:
Oh yes, that would be much better!